src/main.py

```diff
@@ -41,6 +41,10 @@ def main():
             from agents.dqn.train import train_agent
             agent = train_agent(args.hardcore, args.render)
 
+        case 'drq_v2':
+            from agents.drq_v2.train import train_agent
+            agent = train_agent(args.hardcore, args.render)
+
         case 'ppo':
             from agents.ppo.train import train_agent
             agent = train_agent(args.hardcore, args.render)
```

Comment:
Line is repeated, move outside of match case?

Reply:
Done